app.js

```diff
@@ -1,31 +1,74 @@
 import { getUser, signOut } from './services/auth-service.js';
 import { protectPage } from './utils.js';
 import createUser from './components/User.js';
+import createShoppingForm from './components/ShoppingForm.js';
+import { addItem, allItems, updateItem, deleteItem } from './services/list-service.js';
+import createItems from './components/Items.js';
 
 // State
 let user = null;
+let items = [];
 
 // Action Handlers
 async function handlePageLoad() {
     user = getUser();
     protectPage(user);
 
+    items = await allItems();
+    // console.log(data); 
     display();
 }
 
 async function handleSignOut() {
     signOut();
 }
 
+async function handleAdd(item, quantity) {
+    // console.log(item, quantity);
+    const itemsList = { item, quantity, user_id: user.id };
+    const data = await addItem(itemsList);
+
+
+    items.push(data);
+
+    display();
+}
+
+async function handleUpdate(item) {
+    item = !item;
```

Comment:
'item' is an object. Assigning it to !item basically deletes the info from that object, and sets it to null. On this line, you'll want to set the 'bought' key in the item object to its oposite. So, you'll want this line to read
`item.bought = !item.bought`